### PR TITLE
django webapp login with python-social-auth

### PR DESCRIPTION
**Pull request by alexisluque**

### CODE REVIEW

File: WebApp-example/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+#If we add the requirements and install dependencies first, docker can use cache if requirements don't change
+ADD requirements.txt /home/app
+RUN pip install --no-cache-dir -r requirements.txt
+
+ADD . /home/app
+
+# Migrate the database
+RUN python manage.py migrate
+
```
Comment (aaguiarz) [inline WebApp-example/Dockerfile]: Migrate the database?

**Comment by aaguiarz** [comment]
- There should be a single folder called 01-login. 
- What's the webappexample folder?
- Readme.md is missing

File: WebApp-example/webappexample/settings.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+AUTHENTICATION_BACKENDS = {
+    'auth0login.auth0backend.Auth0',
+    'django.contrib.auth.backends.ModelBackend'
+}
+
+
+LOGIN_URL = "/login/auth0"
+LOGIN_REDIRECT_URL = "/dashboard"
+LOGOUT_REDIRECT_URL = "/"
```
Comment (aaguiarz) [inline WebApp-example/webappexample/settings.py]: Do we need all the settings in this file? can we simplify it?

File: WebApp-example/Dockerfile (hunk 1 above)
Comment (alexisluque) [reply WebApp-example/Dockerfile]: Yes, we need to initialize the database (Django defaults to SQLite) to use the built-in Django user authentication system

File: WebApp-example/webappexample/settings.py (hunk 2 above)
Comment (alexisluque) [reply WebApp-example/webappexample/settings.py]: Before SOCIAL_AUTH settings, all is Django default minimun settings autogenerated

**Pull request merged**